### PR TITLE
Change homebrew install instructions

### PR DESCRIPTION
**Pull request by perryprog**
Homebrew now automatically updates for you. I also removed a strange tab (?)

### CODE REVIEW

File: app/views/docs/_install_homebrew.html.md [hunk 1]
```diff
@@ -1 +1 @@
-    brew update && brew install asciinema
+brew install asciinema
```
Comment (sickill) [inline app/views/docs/_install_homebrew.html.md]: Looks like you removed the indentation again...

Comment (perryprog) [inline app/views/docs/_install_homebrew.html.md]: Fixed

**Comment by sickill** [comment]
I've cherry-picked your change into master. Thanks!

**Comment by perryprog** [comment]
K. It's hilarious how if you click on pull requests there are three subsequent contributions from me. 😝

**Pull request closed without merging**